concurrent_test.go

```diff
@@ -511,6 +537,23 @@ func (rs historyRecords) Swap(i, j int) {
 	rs[i], rs[j] = rs[j], rs[i]
 }
 
+func validateIncrementalTxid(rs historyRecords, op OperationType) error {
+	if len(rs) == 0 {
+		return nil
+	}
+
+	lastTxid := rs[0].Txid
+
+	for i := 1; i < len(rs); i++ {
+		if (op == Write && rs[i].Txid <= lastTxid) || (op == Read && rs[i].Txid < lastTxid) {
+			return fmt.Errorf("detected non-incremental txid(%d, %d) in %s mode", lastTxid, rs[i].Txid, op)
+		}
```

Comment:
This mixes testing Sequential property (txid is non-decreasing) and Write increasing txid. Recommend to only test Sequential property, As Write increasing txid is already tested by `validateSerializable`

Reply:
> As Write increasing txid is already tested by `validateSerializable`

This isn't correct, because 
1. we sort all the records in `validateSerializable`;
2. the `rs historyRecords` passed to `validateSerializable` includes both read and write operations.

Also I'd like to cover both read and write operationType in `validateIncrementalTxid` so as to be independent to another validation.